README.md

```diff
@@ -26,13 +83,79 @@ cd ape-chainstack
+
+Create an environment variable with your Chainstack node URL in this format `CHAINSTACK_"NETWORK"_URL=ENDPOINT_URL`; for example:
+
+```sh
+export CHAINSTACK_GOERLI_URL=https://nd-11X-26X-16X.p2pify.com/YOUR_API_KEY
+```
+
+Use the command `ape networks list` to see the networks available:
+
+```sh
+ethereum  (default)                                                                                                                 
+├── mainnet                                                                                                                         
+│   ├── geth  (default)                                                                                                             
+│   └── chainstack                                                                                                                  
+├── ropsten                                                                                                                         
+│   ├── geth  (default)                                                                                                             
+│   └── chainstack                                                                                                                  
+├── kovan                                                                                                                           
+│   └── geth  (default)                                                                                                             
+├── rinkeby                                                                                                                         
+│   ├── geth  (default)                                                                                                             
+│   └── chainstack                                                                                                                  
+├── goerli                                                                                                                          
+│   ├── geth  (default)                                                                                                             
+│   └── chainstack                                                                                                                  
+└── local  (default)                                                                                                                
+    ├── geth                                                                                                                        
+    └── test  (default)        
+```    
+
+Use the `--network` command to connect to your node; for example:
+
+```bash
+ape console --network ethereum:goerli:chainstack
+```
+
+You can also see the complete list of the syntax by trying to connect to a network that does not exist:
 
 ```bash
-ape console --network ethereum:ropsten:chainstack
+ape console --network chainstack
```

Comment:
You can see the complete list by doing `ape networks list`. I think we should document it that way instead because it does not then require putting in an errornous command.

Reply:
My idea with this is to show the user the syntax to use and all the different options. But I'm ok removing that part if you prefer, the part above it shows the `ape networks list` command anyway 😄